### DISCUSSION TITLE
Monorepo deployment error:  404: NOT_FOUND Code: NOT_FOUND

### DISCUSSION THREAD

**Discussion by AndrewNow** [vercel/community#581] [Help] [answered]
Hi all,

My project follows [the following configuration](https://www.sanity.io/guides/building-ecommerce-sites-with-the-stripe-api). At the root, my project has two subdirectories:

`/studio` for Sanity CMS config/schema;
`/web` for all my page files, components, all your standard Next.js stuff.)

When I run the project in my localhost:3000, I cd into /web and run `npm run dev` from there. 

I've also tried setting the Root directory to `web` to no effect.

![image](https://user-images.githubusercontent.com/57265008/165782015-acc13c81-20b3-488f-9e3a-66cc83773adb.png)

How should I go about deploying my project? Here's [the repo.](https://github.com/AndrewNow/Expose-Noir)

**Comment by AndrewNow** [comment, marked as answer]
EDIT: I found the solution, I had to add a `vercel.json` file at the root with the following:
```js
{
  "version": 2,
  "rewrites": [{ "source": "/(.*)", "destination": "/index.html" }]
}
```

**Reply by BarkMulcher** [reply]
I have noted from their project config [docs](https://vercel.com/docs/project-configuration#legacy/version) that we should no longer use "version". I have used your "rewrites" script with no success, though it didn't worsen my issue. Do you know what else could be added to help me correctly deploy with vercel?

**Reply by amyegan** [reply]
Hi @BarkMulcher! If you [start a new discussion](https://github.com/orgs/vercel/discussions/new?category=help) with more details about your situation, then we can work together to find a solution to the specific problem you're facing 🙏
